pkg/collector/corechecks/containers/docker.go

```diff
@@ -236,6 +239,10 @@ func (d *DockerCheck) Run() error {
 		}
 	}
 
+	// Update the container size counter, used to collect them less often as they are costly
+	d.collectContainerSizeCounter =
```

Comment:
Ideally, that would be under the `if d.instance.CollectContainerSize {` branch, no need to update this int if the feature is disabled (which is the default)

Reply:
Indeed! 👍